src/parseTools.js

```diff
@@ -1477,6 +1477,34 @@ function makeStaticString(string) {
   return '(stringToUTF8("' + string + '", ' + ptr + ', ' + len + '), ' + ptr + ')';
 }
 
+// Generates access to module exports variable in pthreads worker.js
```

Comment:
these comments could be better - should explain the stuff we are discussing here. in particular, why pthreads is different.

Reply:
updated the comments with more detailed info